adapters/rise/rise.go

```diff
@@ -0,0 +1,134 @@
+package rise
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"net/http"
+	"strings"
+
+	"github.com/prebid/openrtb/v19/openrtb2"
+	"github.com/prebid/prebid-server/adapters"
+	"github.com/prebid/prebid-server/config"
+	"github.com/prebid/prebid-server/errortypes"
+	"github.com/prebid/prebid-server/openrtb_ext"
+)
+
+// adapter is a Rise implementation of the adapters.Bidder interface.
+type adapter struct {
+	endpointURL string
+}
+
+func Builder(_ openrtb_ext.BidderName, config config.Adapter, _ config.Server) (adapters.Bidder, error) {
+	return &adapter{
+		endpointURL: config.Endpoint,
+	}, nil
+}
+
+// MakeRequests prepares the HTTP requests which should be made to fetch bids.
+func (a *adapter) MakeRequests(
+	openRTBRequest *openrtb2.BidRequest,
+	_ *adapters.ExtraRequestInfo) (
+	requestsToBidder []*adapters.RequestData,
+	errs []error,
+) {
```

Comment:
Nitpick - for consistency in the codebase please use single line for MakeRequests function declaration. 
```
func (a *adapter) MakeRequests(openRTBRequest *openrtb2.BidRequest, _ *adapters.ExtraRequestInfo) (requestsToBidder []*adapters.RequestData, errs []error) {
//your code
}
```